partcad/src/partcad/test/all.py

```
@@ -0,0 +1,27 @@
+
+_global_tests = []
+
+
+def tests() -> list[Test]:
+    if len(_global_tests) == 0:
+        _global_tests.extend(
+            [
+                CadTest(),
+                CamTest(),
+                CamAdditiveSolidTest(),
+            ]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)